main.tf

```
@@ -358,3 +358,10 @@ resource "aws_cloudwatch_event_api_destination" "this" {
   invocation_rate_limit_per_second = lookup(each.value, "invocation_rate_limit_per_second", null)
   connection_arn                   = aws_cloudwatch_event_connection.this[each.value.name].arn
 }
+
+resource "aws_cloudwatch_event_bus_policy" "this" {
+  count = var.policy_event_bus_json != null ? 1 : 0
```

```suggestion
  count = var.create && var.create_event_bus_policy ? 1 : 0
```

Add `create_event_bus_policy` variable, too.